File: sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    height = get_height()
    for i in range(1, height + 1):
        print(" " * (height - i), end="")
        print("#" * i)


def get_height():
    while True:
        n = get_int("Heigth: ")
        if n > 0 and n <= 8:
            break
    return n


main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

// Get heigth
    int h;
    do
    {
        h = get_int("Heigth: ");
    }
    while (h < 1 || h > 8);

    // Print blocks and dots
    for (int i = 1; i <= h; i++)
    {
        for (int j = 1; j <= h - i; j++)
        {
            printf(" ");
        }
        for (int a = 1; a <= i; a++)
        {
            printf("#");
        }
        printf("\n");
    }
}